templates/profile/dns/smartconnect/named.conf.options.erb

```diff
@@ -3,7 +3,8 @@
 options {
   directory "/var/cache/bind";
 
-  dnssec-validation auto;
+  dnssec-enable no;
```

Comment:
I can't imagine this would break anything on stretch since AFAIK we aren't using it anyway, but it would be worth verifying on e.g. test.babel.hathitrust.org before applying in production.

Reply:
I did try this out on `test.babel.hathitrust.org` and verified that the Isilon mounts still work after the change to `named.conf`.